redis/src/commands/mod.rs

```diff
@@ -400,6 +404,51 @@ implement_commands! {
         cmd("HEXISTS").arg(key).arg(field)
     }
 
+    /// Get one or more fields TTL in seconds.
+    fn httl<K: ToRedisArgs, F: ToRedisArgs>(key: K, fields: &'a [F]) {
```

Comment:
I believe that here, and below, `[fields](fields: &'a [F])` can be replaced by `field: F`, since slices implement ToRedisArgs.

Reply:
@nihohit Thanks for your reply. I use slices here since those commands need to know how many fields are sent. So want to know how I can get the length using the `ToRedisArgs`? 

I'm a very newbie to Rust, so forgive me if asked silly questions.

Reply:
you're right, I missed that.

@jaymell do you think we should add to `ToRedisArgs` a length function, instead of `is_single_arg`?

Reply:
@jaymell How do you think about this proposal? I will change it if it sounds good.

Reply:
@jaymell What nihohit means should be to remove `is_single_arg` and replace it with the `len` function.

Reply:
Ok, I guess we could do that, but we should probably put that in a separate PR and get that in first.

Reply:
ok, thanks.